spec/services/pull_request/parse_repository.rb

```diff
@@ -0,0 +1,15 @@
+describe 'PullRequest::ParseRepository' do
+
+  context 'within given repo url' do
+    let(:repo_url) { 'https://github.com/example_company/example_project/pull/99' }
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.